tests/integration/test_data_finder.py

```diff
@@ -65,7 +69,7 @@ def create_tree(path, filenames=None, symlinks=None):
 def test_get_output_file(cfg):
     """Test getting output name for preprocessed files."""
     output_file = get_output_file(cfg['variable'], cfg['preproc_dir'])
-    assert output_file == cfg['output_file']
+    assert [output_file] == cfg['output_file']
```

Comment:
Why this change? get_output_file should return a single filename, right?

Reply:
There was a complain during the test about a line being too long in the `data_finder.yml`, so I moved the filename from this:
    output_file: this/is/a/path/test_diag/test/CMIP5_HadGEM2-ES_Amon_historical_r1i1p1_ta_1960-1980.nc

to this:
    output_file:
      - this/is/a/path/test_diag/test/CMIP5_HadGEM2-ES_Amon_historical_r1i1p1_ta_1960-1980.nc
So get_output_file still returns a file, but what's in the config is a list.

I don't think anything about the functionality changed. But I have to re-write the test anyway.

Reply:
codacy/flake should never complain a line is too long in a yaml file, Saskia, you sure it wasn't in any of the :snake: codes?

Reply:
yamllint does

Reply:
I think this change to the yaml file is confusing, because the function returns a single string, not a list containing a string. If you want to get rid of the yamllint warning, you could [spread the string over two lines](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines) instead.